### ISSUE TITLE
Is wx.html.HtmlPrintout abstract class in wxPython 4?

### ISSUE THREAD

**Issue by Jenyay** [wxWidgets/Phoenix#1060]
**Operating system**: Ubuntu 18.04
**wxPython version & source**:          wxPython 4.0.1
**Python version & source**:            Python 3.6 stock.

I used the wx.html.HtmlPrintout class In wxPython 3.0.2. Is wx.html.HtmlPrintout abstract class in wxPython 4?

I started using HtmlEasyPrinting class instead, but HtmlEasyPrinting makes it impossible to set page margins and page range. Or are there such opportunities?

```python
>>> import wx.html
>>> wx.html.HtmlPrintout()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: wx._html.HtmlPrintout represents a C++ abstract class and cannot be instantiated
```

**Comment by RobinD42** [comment]
No, that's a mistake. I'll add a fix shortly.

**Comment by RobinD42** [comment]
Fixed by #1064
